RecoPixelVertexing/PixelVertexFinding/src/gpuVertexFinder.h

```diff
@@ -0,0 +1,78 @@
+#ifndef RecoPixelVertexing_PixelVertexFinding_gpuVertexFinder_H
+#define RecoPixelVertexing_PixelVertexFinding_gpuVertexFinder_H
+
+#include<cstdint>
+
+#include "RecoPixelVertexing/PixelVertexFinding/interface/pixelVertexHeterogeneousProduct.h"
+
+
+namespace gpuVertexFinder {
+
+  struct OnGPU {
+
+    static constexpr uint32_t MAXTRACKS = 16000;
+    static constexpr uint32_t MAXVTX= 1024;
+    
+    float * zt;
```

Comment:
Can you use some more descriptive names for these variables, rather than `zt`, `zv`, `wv`, `nz`, `iv` ?
Or at least at a comment here mentioning what they are supposed to be ?